blocktx/store/interface.go

```diff
@@ -20,6 +20,8 @@ var (
 
 type Interface interface {
 	RegisterTransaction(ctx context.Context, transaction *blocktx_api.TransactionAndSource) (string, string, []byte, uint64, error)
+	TryToBecomePrimary(ctx context.Context, myHostName string) error
```

Comment:
Please add `//go:generate moq -out ./store/mock.go ./store Interface` to some where peer_handler_test.go so that in the future running `go generate ./...` is enough to recreate the mocks from a change in the interface

Reply:
You mean to add this in Make file?